wordpress-mobile/WordPress-iOS
# Jetpack App (Emphasis): Use dynamic dates in screenshots

Part of #16395
WPMocks: https://github.com/wordpress-mobile/WordPressMocks/pull/31

## Description

This PR updates WordPressMocks.
- Updated the activity log stub and scan stub to use dynamic dates.

Activity Log | Scan | Jetpack Backup
:--: | :--: | :--:
Today and yesterday | Yesterday | Today
![Simulator Screen Shot - iPhone 12 Pro - 2021-06-04 at 17 13 19](https://user-images.githubusercontent.com/6711616/120769572-3c09c800-c558-11eb-8b68-d1b234bb633e.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-06-04 at 17 13 25](https://user-images.githubusercontent.com/6711616/120769576-3d3af500-c558-11eb-93da-f033f14ff637.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-06-04 at 17 13 32](https://user-images.githubusercontent.com/6711616/120769581-3dd38b80-c558-11eb-95e0-a2bd156ccb0f.png)

## How to test
1. Switch to the `JetpackScreenshotGeneration` target
2. Run `rake mocks` to start a local mock server for UI tests
3. ✅ Make sure that `testGenerateScreenshots()` succeeds

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A 

PR submission checklist:
